RedHatInsights/rhsm-subscriptions
# SWATCH-64: Add billing provider to subscription api response

https://issues.redhat.com/browse/SWATCH-64

Test Steps:

- Insert Test Data:
```
INSERT INTO public.offering VALUES ('MW01882', 'OpenShift Streams for Apache Kafka', 'JBoss', NULL, NULL, NULL, NULL, NULL, 'Premium', '', 'Red Hat OpenShift Streams for Apache Kafka (Hourly)', false);
INSERT INTO public.subscription VALUES ('MW01882', 'org123', '2352521', 1, '2011-01-01 01:02:33+00', '2031-01-01 01:02:33+00', 'testProductCode123;customer123;awsSellerAccountId', '123', '4243626', 'red hat', '1234567891234');
INSERT INTO public.subscription VALUES ('MW01882', 'org123', '235252', 1, '2020-01-01 05:00:00+00', '2030-01-01 05:00:00+00', 'testProductCode123;customer123;awsSellerAccountId', '123', '1234', 'aws', '1234567891234');
```
- Start app with - `DEV_MODE=true ./gradlew :bootRun`
- Run curl :
```
curl -X 'GET' \
  'http://localhost:8000/api/rhsm-subscriptions/v1/subscriptions/products/rhosak' \
  -H 'accept: application/vnd.api+json' \
  -H 'x-rh-identity: eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6ImFjY291bnQxMjMiLCJ0eXBlIjoiVXNlciIsInVzZXIiOnsiaXNfb3JnX2FkbWluIjp0cnVlfSwiaW50ZXJuYWwiOnsib3JnX2lkIjoib3JnMTIzIn19fQo='
```
- Should get a row returned for AWS and Red Hat billing providers
